### ISSUE TITLE
Accessibility Audit

### ISSUE THREAD

**Issue by qiqicodes** [hackforla/website#1347]
### Dependency
- [x] https://github.com/hackforla/website/issues/3859
- [ ] https://github.com/hackforla/website/issues/4300 
   - [ ] and whatever issues are contained in the above epic are closed
- [ ] check the board, and if there doesn't seem to be any accessibility epics that should be done first, this can be moved to new issue approval.
  - [Accessibility issues on the board](https://github.com/orgs/hackforla/projects/86/views/1?filterQuery=-label%3A%22away+on+hold%22+label%3A%22Feature%3A+Accessibility%22) 

### Overview
Accessibility Audit to follow the WCAG 2.0/2.1 guidelines 

### Action Items
- [ ] Alt text for informative and decorative images
- [ ] Text proportion and color contrast
- [ ] Link labels
- [ ] Video/Gif captions
- [ ] Info & Relationship 
        - [ ] Improve current project card’s headings and links’ descriptions
- [ ] Meaning sequence audit 
        - [ ] test with keyboard tab to see focusable elements
        - [ ] test with a blur filter along side with screen reader and voiceover 
- [ ] Labels and Instruction for Toolkit Guides

### Resources/Instructions
**Guidelines/Law:**
- WCAG: 
        - [WCAG 2.0 & 2.1](https://www.w3.org/WAI/standards-guidelines/wcag/)
        - [Quick Reference](https://www.w3.org/WAI/WCAG21/quickref/)
- 508 Compliance
        - [Tools & Guidance](https://www.section508.gov/test/web-software)

**Tool Extensions:**

- [WAVE](https://wave.webaim.org/extension/)
- [Siteimprove Accessibility Checker](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc)

Here are some of the sample errors we currently have for Hack for LA website:
- [Hack for LA Web Accessibility Results ](https://github.com/hackforla/website/files/6246294/HackforLA.Results.-.Web.Accessibility.pdf)

- ![Hack for LA Home Wave Extension Details](https://user-images.githubusercontent.com/68657634/113349277-a39f5f00-92ec-11eb-8ce2-ff5389ccaba1.png)

- ![Hack for LA Home Siteimprove Accessibility Checker Extension](https://user-images.githubusercontent.com/68657634/113349411-d8abb180-92ec-11eb-9ed6-2bf5f7716c65.png)

**Comment by ExperimentsInHonesty** [comment]
### Overview
There are more than 100 accessibility testing tools.  Figuring out which ones to use can be a black hole.  For guidance we recommend this article:  [Which accessibility testing tool should you use?](https://medium.com/pulsar/which-accessibility-testing-tool-should-you-use-e5990e6ef0a)

### Summary of Article
The author recommends using the tools in the following order fixing as you go along, since no one tool catches all the relevant issues

[aXe](https://www.deque.com/axe/)
[SiteImprove](https://chrome.google.com/webstore/detail/siteimprove-accessibility/efcfolpjihicnikpmhnmphjhhpiclljc)
[Tenon](https://tenon.io/)
[WAVE](https://wave.webaim.org/extension/)
[Lighthouse](https://developers.google.com/web/tools/lighthouse/#devtools)

But if you want to test your site with other tools, here is a bigger [list](https://www.w3.org/WAI/ER/tools/)

**Comment by ExperimentsInHonesty** [comment]
The review could be done now and just ignore the alt text flags.

**Comment by ExperimentsInHonesty** [comment]
We are going to make one issue using this template

https://github.com/hackforla/.github/issues/new?assignees=&labels=&template=wave-chrome-extension--accessibility-review.md&title=Wave+Chrome+Extension%3A+Accessibility+review

Ask QiQi how often we should do it 

Checking the board to see if the feasibility is good (meaning there are either not a lot of outstanding issues or they are all of a specific type).

**Comment by adam-abundis** [comment]
@GLRJr brought up a good point for using an accessibility linter. We could use a [vs code extension from deque systems like this](https://marketplace.visualstudio.com/items?itemName=deque-systems.vscode-axe-linter)

**Comment by Aveline-art** [comment]
@qiqicodes Can you talk to us about how often an accessibility audit should be made, in your opinion? So far, we plan to perform an audit after the current slew of accessibility issues are completed. Afterwards, should an audit be performed at a regular interval? Let me know, because if so, we can create a GHA to create an automated issue.
